log_util_abort failed

`log_util_abort` failed in komodo with:
```
test_log_util_abort ______________________________

caplog = <_pytest.logging.LogCaptureFixture object at 0x7f429187feb8>
monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0x7f429187fc88>

    def test_log_util_abort(caplog, monkeypatch):
        shutdown_mock = MagicMock()
        monkeypatch.setattr(logging, "shutdown", shutdown_mock)
        _log_util_abort("fname", 1, "some_func", "err_message", "my_backtrace")
>       assert (
            "C trace:\nmy_backtrace \nwith message: err_message \nfrom file: "
            "fname in some_func at line: 1\n\nPython backtrace:"
        ) in caplog.text
E       AssertionError: assert 'C trace:\nmy_backtrace \nwith message: err_message \nfrom file: fname in some_func at line: 1\n\nPython backtrace:' in ''
E        +  where '' = <_pytest.logging.LogCaptureFixture object at 0x7f429187feb8>.text

/some/path/ert/test-root/tests/ert_tests/shared/test_log_abort.py:10: AssertionError
```

For some reason the caplog is empty after the test has completed.

--- a/tests/ert_tests/shared/test_log_abort.py
+++ b/tests/ert_tests/shared/test_log_abort.py
@@ -6,7 +6,8 @@ from ert_logging._log_util_abort import _log_util_abort
 def test_log_util_abort(caplog, monkeypatch):
     shutdown_mock = MagicMock()
     monkeypatch.setattr(logging, "shutdown", shutdown_mock)
-    _log_util_abort("fname", 1, "some_func", "err_message", "my_backtrace")
+    with caplog.at_level(logging.ERROR):
+        _log_util_abort("fname", 1, "some_func", "err_message", "my_backtrace")
     assert (
         "C trace:\nmy_backtrace \nwith message: err_message \nfrom file: "
         "fname in some_func at line: 1\n\nPython backtrace:"

